doc/api/process.markdown

```diff
@@ -1070,18 +1080,36 @@ Will print something like:
   openssl: '1.0.1k' }
 ```
 
+[`'finish'`]: stream.html#stream_event_finish
 [`'message'`]: child_process.html#child_process_event_message
+[`'rejectionHandled'`]: #process_event_rejectionhandled
+[`'uncaughtException'`]: #process_event_uncaughtexception
 [`ChildProcess.disconnect()`]: child_process.html#child_process_child_disconnect
+[`ChildProcess.kill()`]: child_process.html#child_process_child_kill_signal
 [`ChildProcess.send()`]: child_process.html#child_process_child_send_message_sendhandle_callback
+[`ChildProcess`]: child_process.html#child_process_class_childprocess
+[`end()`]: stream.html#stream_writable_end_chunk_encoding_callback
 [`Error`]: errors.html#errors_class_error
 [`EventEmitter`]: events.html#events_class_events_eventemitter
+[`getegid(2)`]: http://linux.die.net/man/2/getegid
+[`geteuid(2)`]: http://linux.die.net/man/2/geteuid
+[`getgid(2)`]: http://linux.die.net/man/2/getgid
+[`getuid(2)`]: http://linux.die.net/man/2/getuid
+[`kill(2)`]: http://linux.die.net/man/2/kill
 [`net.Server`]: net.html#net_class_net_server
 [`net.Socket`]: net.html#net_class_net_socket
+[`process.argv`]: #process_process_argv
+[`process.disconnect()`]: #process_process_disconnect
 [`process.exit()`]: #process_process_exit_code
-[`promise.catch(...)`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch
-[`'rejectionHandled'`]: #process_event_rejectionhandled
+[`process.kill()`]: #process_process_kill_pid_signal
+[`promise.catch()`]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/catch
 [`require.main`]: modules.html#modules_accessing_the_main_module
+[`setegid(2)`]: http://linux.die.net/man/2/setegid
+[`seteuid(2)`]: http://linux.die.net/man/2/seteuid
+[`setgid(2)`]: http://linux.die.net/man/2/setgid
 [`setTimeout(fn, 0)`]: timers.html#timers_settimeout_callback_delay_arg
+[`setuid(2)`]: http://linux.die.net/man/2/setuid
+[`sigaction(2)`]: http://linux.die.net/man/2/sigaction
```

Comment:
Me too. I'll cut this out, so once 5073 lands, this will be automated. The thing is I did this with script too )